src/render/opengles2/SDL_render_gles2.c

```diff
@@ -1973,6 +2001,7 @@ GLES2_CreateRenderer(SDL_Window *window, Uint32 flags)
 #if SDL_HAVE_YUV
     renderer->UpdateTextureYUV    = GLES2_UpdateTextureYUV;
     renderer->UpdateTextureNV     = GLES2_UpdateTextureNV;
+    renderer->UpdateTextureOES    = GLES2_UpdateTextureOES;
```

Comment:
same (Why is this guarded by SDL_HAVE_YUV ?)

Reply:
I'll fix it